src/adapter/src/coord.rs

```diff
@@ -1467,6 +1484,45 @@ impl<S: Append + 'static> Coordinator<S> {
         tx.send(result, session);
     }
 
+    async fn message_create_connection_plan_ready(
+        &mut self,
+        CreateConnectionPlanReady {
+            mut session,
+            tx,
+            result,
+            params,
+            depends_on,
+            original_stmt,
+        }: CreateConnectionPlanReady,
+    ) {
+        let plan = match result {
+            Ok(stmt) => stmt,
+            Err(e) => return tx.send(Err(e.into()), session),
```

Comment:
Since this function has an empty return signature the `return` here is not obvious. Can you split it into `{ tx.send(..); return; }`.